rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/240_source_synthetic_vectors.yml

```diff
@@ -0,0 +1,151 @@
+setup:
+  - requires:
+      reason: 'synthetic vectors are required'
+      test_runner_features: [ capabilities ]
+      capabilities:
+        - method: GET
+          path: /_search
+          capabilities: [ synthetic_vectors_setting ]
```

Comment:
I *THINK* you need to enable the feature flag for testing. Let me find the correct place.

Reply:
thanks

Reply:
https://github.com/elastic/elasticsearch/pull/130382/commits/62c98c34eeb6af4638bc2ed93b8b299b9fef4e78